nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -66,6 +67,8 @@ public class NullAwaySerializationTest extends NullAwayTestsBase {
   private static final String FIELD_INIT_FILE_NAME = "field_init.tsv";
   private static final String FIELD_INIT_HEADER = FieldInitializationInfo.header();
 
+  private static final int SERIALIZATION_VERSION = 1;
```

Comment:
We should not be declaring this version number in two places.  Just make the other static field `public`